py_gtfs_rt_ingestion/ingest.py

```diff
@@ -157,6 +150,7 @@ def batch_and_ingest() -> None:
 
 def main() -> None:
     """every second run jobs that are currently pending"""
+    batch_and_ingest()
```

Comment:
kick off a batch and ingest and then schedule one to run 5 minutes past the hour every hour.